pyaptly/cli.py

```diff
@@ -19,12 +19,18 @@ def entry_point():
     if len_argv > 0 and argv[0].endswith("pyaptly"):
         if len_argv > 2 and argv[1] == "legacy" and argv[2] != "--":
             argv = argv[:2] + ["--"] + argv[2:]
+    if "--debug" in argv or "-d" in argv:
```

Comment:
minor: it's of course a matter of preference but you could make this a one liner: `debug = "--debug" in argv or "-d" in argv`